### PR TITLE
Czech language

### PR DESCRIPTION
**Pull request by panmlg**

### CODE REVIEW

**Comment by mrjvs** [comment]
please see the code style annotations. (click "files changed" tab on this page)

**Comment by panmlg** [comment]
> please see the code style annotations. (click "files changed" tab on this page)

I apologize, didn't see that. Hopefully it's better now.

**Comment by mrjvs** [comment]
code style is good now, thanks. I'll verify the translations later today.

File: src/setup/locales/cs/translation.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    "stopEditing": "Zastavit upravování",
+    "errors": {
+      "genericTitle": "Jejda, rozbilo se to!",
+      "failedMeta": "Nepovedlo se načíst meta",
+      "mediaFailed": "Nepodařilo se nám požádat o Vaše média, zkontrolujte své internetové připojení a zkuste to znovu.",
+      "videoFailed": "Při přehrávání požadovaného videa došlo k chybě. Pokud se tohle opakuje prosím nahlašte nám to na <0>Discord serveru</0> nebo na <1>GitHubu</1>."
+    }
+  },
+  "seasons": {
+    "seasonAndEpisode": "S{{sezóna}} E{{epizoda}}"
```
Comment (JamesHawkinss) [inline src/setup/locales/cs/translation.json:30]:
```suggestion
    "seasonAndEpisode": "S{{season}} E{{episode}}"
```

**Review by JamesHawkinss — CHANGES_REQUESTED** [review]
Don't translate the keys!

**Review by JamesHawkinss — APPROVED** [review]

**Pull request merged**